ops/storage-ddls/cassandra-setup.cql

```diff
@@ -175,27 +175,27 @@ CREATE TABLE popularplaces (
 
 DROP TABLE IF EXISTS eventtopics;
 CREATE TABLE eventtopics(
-    eventids set<text>,
+    eventid text,
     topic text,
-    insertiontime timestamp,
+    eventtime timestamp,
     pipelinekey text,
     externalsourceid text,
-    PRIMARY KEY ((topic), pipelinekey, externalsourceid)
+    PRIMARY KEY ((topic), pipelinekey, externalsourceid, eventtime, eventid)
 );
 
 DROP TABLE IF EXISTS eventplaces;
 CREATE TABLE eventplaces(
-    eventids set<text>,
+    eventid text,
     conjunctiontopic1 text,
     conjunctiontopic2 text,
     conjunctiontopic3 text,
     centroidlat double,
     centroidlon double,
     placeid text,
-    insertiontime timestamp,
+    eventtime timestamp,
     pipelinekey text,
     externalsourceid text,
-    PRIMARY KEY ((placeid), conjunctiontopic1, conjunctiontopic2, conjunctiontopic3, pipelinekey, externalsourceid, centroidlat, centroidlon)
+    PRIMARY KEY ((placeid, conjunctiontopic1, conjunctiontopic2, conjunctiontopic3), pipelinekey, externalsourceid, eventtime, eventid)
```

Comment:
Not sure I understand the comments, could you elaborate?

We're querying this table in two places:

*Messages.byBbox*
<img src="https://user-images.githubusercontent.com/1086421/29181623-b53a6264-7db0-11e7-9e8f-e9403a79fc86.png" height=200 />

*Messages.byLocation*
<img src="https://user-images.githubusercontent.com/1086421/29181651-cdc99e8a-7db0-11e7-991e-4b926b772fff.png" height=200 />

In both cases we query on the placeid, not on the centroid locations.